src/components/EncounterNotes/MatthiasNotes.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import RoleEmbed from "../RoleEmbed/RoleEmbed";
+import GW2Skill from "../GW2Skill/GW2Skill";
+
+function MatthiasNotes() {
+  return (
+    <div>
+      <h3>Individual Jobs</h3>
+      <ul>
+        <li>
+          <RoleEmbed role="Cnd" profession="mirage" /> brings
```

Comment:
This probably needs updating to use Firebrands.. but this is fine for now.. at least until we've decided who's playing what.